2019/02/24.md

```diff
@@ -0,0 +1,52 @@
+# Redux conventions
+
+Based on the discussion about [prefixing action
+types](https://github.com/dhis2/notes/issues/18) some conventions can be
+extracted for how to use Redux.
+
+Austin mentioned an article about [Redux best
+practices](https://medium.com/@kylpo/redux-best-practices-eef55a20cc72)
+which we can use as inspiration.
+
+## Utilities
+
+- Use [redux-actions](https://github.com/redux-utilities/redux-actions)
+  to create and handle actions
+- Use [redux-thunk](https://github.com/reduxjs/redux-thunk) to deal with
+  logic that has side-effects (async, api, etc.)
+- Use [reselect](https://github.com/reduxjs/reselect) for memoized
+  selectors
+- Use [normalizr](https://github.com/paularmstrong/normalizr) for
```

Comment:
I think this is a bit confusing and needs further clarification.

Kylpo's article on medium.com stated:
> Normalize your data for better reducer composition

Which means: normalize data from api responses to be able to create domain specific reducers instead of one reducer storing the whole response as it's state.
It doesn't mean that reducers should be in any composed with the help of the normalize library!

Reply:
Thanks, I'll add the clarification.